src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentation.cs

```diff
@@ -31,40 +29,41 @@ namespace OpenTelemetry.Instrumentation.StackExchangeRedis
     /// </summary>
     public class StackExchangeRedisCallsInstrumentation : IDisposable
```

Comment:
Can this be made internal now? I made some other instrumentation classes internal in #766, but didn't touch the Redis instrumentation since `StackExchangeRedisCallsInstrumentation` was previously being used in the sample app. It's also mentioned in the README: https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Instrumentation.StackExchangeRedis/README.md.

Reply:
Updated